NEW_README.md

```diff
@@ -4,6 +4,30 @@
 [![Godoc](https://img.shields.io/badge/go-docs-blue?logo=go&logoColor=white&label=%20&labelColor=gray&color=blue)](https://amonks.github.io/run)
 [![Go Report Card](https://goreportcard.com/badge/github.com/amonks/run)](https://goreportcard.com/report/github.com/amonks/run)
 
-Run is a task runner that simplifies executing and managing tasks defined in `tasks.toml` files, with a focus on development workflows. It features an interactive UI for long-lived processes and a non-interactive output mode for short-lived tasks, making it versatile for a wide range of development tasks.
+RUN is a task runner that simplifies executing and managing tasks defined in `tasks.toml` files. It provides a versatile set of features making it well suited for a range of use cases, from simple build scripts to complex development workflows.
```

Comment:
this commit makes this paragraph worse imo (it now only makes vague claims) but it makes the _readme_ better by adding the specific non-vague claims under "features".

I think this tells us that the intro paragraph should be very short, and a features list is a better way to communicate about run. I _love_ the features list. It 100% for sure survives this review.